includes/class-amp-theme-support.php

```diff
@@ -506,13 +506,25 @@ public static function get_amp_component_scripts() {
 	 * Start output buffering.
 	 */
 	public static function start_output_buffering() {
-		ob_start( array( __CLASS__, 'finish_output_buffering' ) );
+		ob_start( array( __CLASS__, 'finish_buffer_add_header' ) );
```

Comment:
We can leave this as `finish_output_buffering` I think.

Reply:
Thanks, this [commit](https://github.com/Automattic/amp-wp/pull/912/commits/83dac8735704fe3d0e0c4e1dc8e2e97d33785ffa) changes the name back to `finish_output_buffering()`.